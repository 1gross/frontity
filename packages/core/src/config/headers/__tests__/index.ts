import getHeaders from "../";

test("Babel returns for development", () => {
  expect(getHeaders({ mode: "development" })).toMatchSnapshot();
});

test("Babel returns for production", () => {
  expect(getHeaders({ mode: "production" })).toMatchSnapshot();
});
