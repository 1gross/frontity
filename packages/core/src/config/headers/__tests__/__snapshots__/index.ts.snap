// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Babel returns for development 1`] = `
Object {
  "dynamic": Object {
    "Access-Control-Allow-Origin": "*",
    "Cache-Control": "no-cache, no-store, must-revalidate",
    "Origin, X-Requested-With, Content-Type": "Accept",
  },
  "static": Object {
    "Access-Control-Allow-Origin": "*",
    "Cache-Control": "no-cache, no-store, must-revalidate",
    "Origin, X-Requested-With, Content-Type": "Accept",
  },
}
`;

exports[`Babel returns for production 1`] = `
Object {
  "dynamic": Object {
    "Access-Control-Allow-Origin": "*",
    "Cache-Control": "public, max-age=0, s-maxage=120, stale-while-revalidate=31536000, stale-if-error=31536000",
    "Origin, X-Requested-With, Content-Type": "Accept",
  },
  "static": Object {
    "Access-Control-Allow-Origin": "*",
    "Cache-Control": "public, max-age=31536000, s-maxage=31536000, immutable",
    "Origin, X-Requested-With, Content-Type": "Accept",
  },
}
`;
